src/field336.json

```diff
@@ -0,0 +1,27 @@
+[
```

Comment:
Move to field33XUtils and export as constant
`export const f336Variables = [..]`

Reply:
Fixed

Reply:
Should we remove this file when these are in field33XUtils?

Reply:
Done